projects/robots/pal_robotics/tiagopp/worlds/tiago++.wbt

```diff
@@ -28,14 +28,14 @@ EXTERNPROTO "webots://projects/objects/traffic/protos/TrafficCone.proto"
 
 WorldInfo {
   info [
-    "This simulation shows a Tiagopp making hello with both arms in an industrial environment. It can also be controlled using the arrows on the keyboard."
+    "This simulation shows a TIAGo ++ making hello with both arms in an industrial environment. It can also be controlled using the arrows on the keyboard."
```

Comment:
```suggestion
    "This simulation shows a TIAGo++ making hello with both arms in an industrial environment. It can also be controlled using the arrows on the keyboard."
```